cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestBase.java

```diff
@@ -101,10 +199,9 @@ protected String getInstanceURI() {
 
   /**
    * CDAP access token for making requests to secure CDAP instances.
-   * Can be obtained via the CDAP authentication server.
    */
-  protected String getAccessToken() {
-    return System.getProperty("accessToken", "");
+  private AccessToken getAccessToken() {
```

Comment:
on second thoughts, it can be useful to have this as public, since ClientConfig builder is public as well.